### PR TITLE
Added new action named Change_Attribute_Value

### PR DESCRIPTION
**Pull request by Towshik**
## PR Type

Added a new feature
PR_TYPE

## PR Checklist

- [ ] Tests for the changes have been added / updated.
- [x] Documentation comments have been added / updated.
- [ ] A changelog entry has been made.
- [ ] Version number has been updated.
- [ ] Required modules have been added to respective "requirements*.txt" files.
- [x] Relevant Test Cases added to this description (below).
- [ ] (Team) Label with affected action categories and server status.

## Overview

Added a new action called Change_Attribute_Value. This action lets the user change the value of any html attributes such as id, placeholder, tag etc (except class). using a value of user's liking. This change in value will help in automation as the user can identify an attribute using necessary locating elements and then change the value of the attribute to run more automation work under one session.

## Test Cases
- [TEST-5648](https://qa.zeuz.ai/Home/ManageTestCases/Edit/TEST-5648/)

### CODE REVIEW

File: Framework/Built_In_Automation/Web/Selenium/BuiltInFunctions.py [hunk 1]
```diff
@@ -1404,6 +1404,29 @@ def take_screenshot_selenium(data_set):
...
+            left = left.strip().lower()
+            if "input parameter" in mid:
+                attribute_name = left
+                change_value = right
+
+        selenium_driver.execute_script(f"arguments[0].{attribute_name} = `{change_value}`;", Element)
+        CommonUtil.ExecLog(sModuleInfo, "Successfully set the value of to text to: %s" % change_value, 1)
+        return "passed"
+    except Exception:
+        errMsg = "Could not select/click your element."
```
Comment (Muntasib-creator) [inline Framework/Built_In_Automation/Web/Selenium/BuiltInFunctions.py]: Wrong error message and passed message

Comment (Muntasib-creator) [reply Framework/Built_In_Automation/Web/Selenium/BuiltInFunctions.py]: @Towshik

**Review by Muntasib-creator — CHANGES_REQUESTED** [review]

File: Framework/Built_In_Automation/Web/Selenium/BuiltInFunctions.py (hunk 1 above)
Comment (Towshik) [reply Framework/Built_In_Automation/Web/Selenium/BuiltInFunctions.py]: Fixed the messages

**Comment by sazid** [comment]
@Towshik fix the pr description.

**Comment by Towshik** [comment]
> @Towshik fix the pr description.

Done vai

**Review by Muntasib-creator — APPROVED** [review]

**Pull request merged**